packages/dashboard-core-plugins/src/linker/LinkerUtils.tsx

```diff
@@ -38,6 +38,8 @@ export type LinkDataMapValue = {
   type: string;
   text: string;
   value: string;
```

Comment:
I think we need to update this - the value isn't always a `string`, this should be `unknown`. Then you don't need that ugly casting in `Linker.tsx`.